src/core/include/openvino/core/rtti.hpp

```diff
@@ -6,26 +6,36 @@
 
 #include "openvino/core/type.hpp"
 
+#if defined _WIN32 || defined __CYGWIN__
+#    define _OPENVINO_HIDDEN_METHOD
+#else
+#    if __GNUC__ >= 4
+#        define _OPENVINO_HIDDEN_METHOD __attribute__((visibility("hidden")))
+#    else
+#        define _OPENVINO_HIDDEN_METHOD
+#    endif
+#endif
```

Comment:
I suppose we need to put these definitions there https://github.com/openvinotoolkit/openvino/blob/master/src/core/include/openvino/core/visibility.hpp#L37-L40

Reply:
Thanks, will be moved

Reply:
Moved